src/PresentationalComponents/Header/Header.test.js

```diff
@@ -1,32 +1,33 @@
 /* eslint-disable */
-import toJson from 'enzyme-to-json';
 import React from 'react';
-import { BrowserRouter as Router } from 'react-router-dom';
+import { BrowserRouter as Router} from 'react-router-dom';
 import Header from './Header';
+import { render } from '@testing-library/react';
+
 
 const withLink = Component => {
```

Comment:
We should rename this to something more fitting like `Wrapper`. Additionally, i'd not call the `render` here but separately in each test case.

Reply:
by renaming this to something more fitting - you mean to create a function for wrapping the components like here? 
https://testing-library.com/docs/react-testing-library/setup/

Reply:
I mean this should be something named similar to this wrapper here: https://github.com/RedHatInsights/insights-inventory-frontend/pull/2033/files#diff-a24acf51249c422f4ce15987438346f5ef701213be1e12c72125dcf53832deccR13

Also note the move of `render` out of a function like this.